Current HEAD misreads fares in the network

Tried running 2018 with the newest olusanya HEAD but `helmet_validate_inputfiles.py` gives a false error on fares in the 2018 network.

Below # 0 is 2040ve2 (ABCD zones in both network and zone data) and # 1 is 2018 (municipality zones in both network and zone data). However, `helmet_validate_inputfiles.py` states that both networks have same fares.

```
2023-04-13 17:24:45 [INFO] Checking base inputdata...
2023-04-13 17:25:13 [INFO] Checking base zonedata & scenario-based input data...
2023-04-13 17:25:13 [INFO] Checking input data for scenario #0 ...
2023-04-13 17:25:23 [DEBUG] Zonedata has fare zones C, M, D, A, N, P, J, B, V
2023-04-13 17:25:23 [DEBUG] Network has fare zones C, M, 0000, D, A, N, P, J, O, B, V
2023-04-13 17:25:24 [INFO] Checking input data for scenario #1 ...
2023-04-13 17:25:35 [DEBUG] Zonedata has fare zones X, L, M, N, E, P, J, H, V
2023-04-13 17:25:35 [DEBUG] Network has fare zones C, M, 0000, D, A, N, P, J, O, B, V
2023-04-13 17:25:35 [ERROR] Found transit fare zone for only 12% of nodes.
```

@zptro, is this an ill side effect from one of your PRs? I can look into this if you don't have time.

--- a/Scripts/helmet_validate_inputfiles.py
+++ b/Scripts/helmet_validate_inputfiles.py
@@ -156,6 +156,9 @@ def main(args):
                     attr_space)
                 log.error(msg)
                 raise ValueError(msg)
+            for scen in emmebank.scenarios():
+                if scen.zone_numbers != zone_numbers:
+                    log.warn("Scenarios with different zones found in EMME bank!")
             scen = emmebank.scenario(first_scenario_ids[i])
             if scen is None:
                 msg = "Project {} has no scenario {}".format(emp_path, scen.id)
@@ -166,9 +169,6 @@ def main(args):
                     scen.id)
                 log.error(msg)
                 raise ValueError(msg)
-            for scen in emmebank.scenarios():
-                if scen.zone_numbers != zone_numbers:
-                    log.warn("Scenarios with different zones found in EMME bank!")
             for attr in link_attrs + line_attrs:
                 if scen.extra_attribute(attr) is None:
                     msg = "Extra attribute {} missing from scenario {}".format(

